modules/data-streams/src/main/java/org/elasticsearch/datastreams/action/TransportGetDataStreamsAction.java

```diff
@@ -78,11 +84,12 @@ public GetDataStreamsTransportAction(
             GetDataStreamAction.Request::new,
             indexNameExpressionResolver,
             GetDataStreamAction.Response::new,
-            EsExecutors.DIRECT_EXECUTOR_SERVICE
+            transportService.getThreadPool().executor(ThreadPool.Names.MANAGEMENT)
```

Comment:
Is this the right thread pool? I don't see one that looks more relevant, but just wanted to make sure that this isn't going to slow down ordinary data stream requests since the comment on the pool says `Not a latency sensitive thread pool: some tasks may time be long-running; and the thread pool size is limited / relatively small.`

Reply:
This is the threadpool that the data stream stats API uses, and since I'm making this API more "heavy" by doing the fan-out-to-shards piece when `verbose` is used, I figured it would be good to get it off of the transport thread